network-api/networkapi/buyersguide/factory.py

```diff
@@ -75,151 +64,6 @@ class Meta:
     snippet = Faker('sentence')
 
 
-"""
```

Comment:
this might as well go, if I'm touching this file anyway. It does nothing, and keeping it sure as heck is dirty.